src/storage/mvcc/txn.rs

```diff
@@ -734,7 +774,11 @@ impl<S: Snapshot, P: PdClient + 'static> MvccTxn<S, P> {
                 )
             }
```

Comment:
Is it possible that there is already a rollback with the same commit_ts?

Reply:
According to our discussion in our slack channel, it's very likely to be impossible. Maybe it should be proved by TLA+ later.

Reply:
Ah sorry I didn't notice the discussion yesterday